### PR TITLE
Ml folder/folder pagination

### PR DESCRIPTION
**Pull request by petersg83**
⚠️  to merge after #13312

Remove pagination for folders

### CODE REVIEW

File: packages/core/strapi/lib/services/entity-service/index.d.ts [hunk 1]
```diff
@@ -54,14 +55,19 @@ export interface EntityService {
     pagination: PaginationInfo;
   }>;
 
-  findWithRelationCounts<K extends keyof AllTypes, T extends AllTypes[K]>(
+  findWithRelationCountsPage<K extends keyof AllTypes, T extends AllTypes[K]>(
     uid: K,
     params: Params<T>
   ): Promise<{
     results: T[];
```
Comment (gu-stav) [inline packages/core/strapi/lib/services/entity-service/index.d.ts:62]: Should this become `data: T[];`

File: packages/core/strapi/lib/services/entity-service/index.d.ts [hunk 2]
```diff
@@ -54,14 +55,19 @@ export interface EntityService {
...
     pagination: PaginationInfo;
```
Comment (gu-stav) [inline packages/core/strapi/lib/services/entity-service/index.d.ts:63]: Should this be removed?

**Review by gu-stav — COMMENTED** [review]
Good work (again).

I'm not sure about the name of the method, but don't have a better suggestion on top of my head. Does @Convly have an idea?

File: packages/core/strapi/lib/services/entity-service/index.d.ts (hunk 1 above)
Comment (petersg83) [reply packages/core/strapi/lib/services/entity-service/index.d.ts:62]: I don't think so, it's not the route response but the function response so there isn't the format `data`.
Are you referring to the function `findWithRelationCounts` line 66?

What I did was:
- Renaming `findWithRelationCounts` to `findWithRelationCountsPage` as it was using the pagination mode (like `findPage` is the equivalent of `findMany` but with pagination) but didn't have this information in the function name.
- Creating the function `findWithRelationCounts` that doesn't use the pagination mode

File: packages/core/strapi/lib/services/entity-service/index.d.ts (hunk 2 above)
Comment (petersg83) [reply packages/core/strapi/lib/services/entity-service/index.d.ts:63]: It was removed in the other function (see comment above)

**Review by gu-stav — APPROVED** [review]

**Pull request merged**